tgui/packages/tgui/interfaces/SpawnPanel/types.ts

```diff
@@ -0,0 +1,59 @@
+import { BooleanLike } from 'tgui-core/react';
+
+export interface Data {
+  [panelName: string]: CreateObjectData[];
+}
```

Comment:
1. why is this an interface and the rest types, or why are those types and everywhere else an interface? be consistent please
2. `Data` is best served for local definitions of backend state only, exporting them is confusing (they show up in code actions now)
3. where is this used?
4. use the `Record<string, CreateObjectData[]>` utility type